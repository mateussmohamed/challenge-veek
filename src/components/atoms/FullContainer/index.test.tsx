import React from 'react'

import { FullContainer } from './index'
import withThemeProvider from '../../theme/withThemeProvider'

describe('[<FullContainer />]', () => {
  const wrap = props => withThemeProvider(() => <FullContainer {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a container!' })

    expect(wrapper).toMatchSnapshot()
  })
})
