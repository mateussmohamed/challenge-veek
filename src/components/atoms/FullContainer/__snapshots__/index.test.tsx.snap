// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<FullContainer />] matches to snapshot 1`] = `[Function]`;
